lib/ansible/template/__init__.py

```diff
@@ -100,11 +100,11 @@ def generate_ansible_template_vars(path, fullpath=None, dest_path=None):
 
     managed_default = C.DEFAULT_MANAGED_STR
     managed_str = managed_default.format(
-        host=temp_vars['template_host'],
-        uid=temp_vars['template_uid'],
-        file=temp_vars['template_path'].replace('%', '%%'),
+        host="{{ template_host }}",
+        uid="{{ template_uid }}",
+        file="{{ template_path.replace('%', '%%') }}"
```

Comment:
This replacement shouldn't be done, since `strftime` will only see the template before it's rendered:

```suggestion
        file="{{ template_path }}"
```